docs/source/guides/statistical_insights.ipynb

```diff
@@ -209,6 +209,209 @@
    "source": [
     "With the normally distributed set of data, the likelyhood of outliers is closer to what we'd expect, around .7%. "
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Inferring Frequency from Noisy Timeseries Data\n",
+    "\n",
+    "`df.ww.infer_temporal_frequencies` will infer the observation frequency (daily, biweekly, yearly, etc) of each temporal column, even on noisy data. If a temporal column is predominantly of a single frequency, but is noisy in any way (ie. contains duplicate timestamps, nans, gaps, or timestamps that do not align with overall frequency), this table accessor will provide the most likely frequency, and the rows of data that do not adhere to this frequency."
```

Comment:
Suggestion for a slight wording change here from 

> this table accessor will provide the most likely frequency, and the rows of data that do not adhere to this frequency.

to

> this table accessor method will provide the most likely frequency as well as information about the rows of data that do not adhere to this frequency.

Reply:
done